AElf.Contracts.TestBase/ContractTester.cs

```diff
@@ -303,7 +317,7 @@ private MinerService BuildMinerService(List<Transaction> txs, List<Transaction>
 
             return new MinerService(mockTxHub.Object, _accountService, _blockGenerationService,
```

Comment:
miner service should not depend on a tx Hub

maybe minerService has miner, miner can have a tx hub

Reply:
TxHub needs to be retrieved from IChainRelatedComponentManager<ITxHub>, can MinerService depend on IChainRelatedComponentManager<ITxHub>?